build-osx.sh

```diff
@@ -148,7 +178,7 @@ run_clean_builds()
 run_clean_all()
 {
     run_clean_builds
-    rm -rf Surge.xcworkspace *xcodeproj target product
+    rm -rf Surge.xcworkspace *xcodeproj target products build_logs
```

Comment:
If wanted to be thorough should remove `obj` as well?

Reply:
Oh yeah. I'll add that in and merge. Thanks!